src/DocumentsRepository/Traits/ExecutableQueryDocumentsRepositoryTrait.ts

```diff
@@ -1,15 +1,30 @@
 import { ExecutableQueryDocument } from "../../ExecutableQueryDocument/ExecutableQueryDocument";
 import { GETOptions, RequestOptions } from "../../HTTP/Request";
+import { Response } from "../../HTTP/Response";
 import { ModelDecorator } from "../../Model/ModelDecorator";
 import { ModelPrototype } from "../../Model/ModelPrototype";
 import { QueryDocumentBuilder } from "../../QueryDocuments/QueryDocumentBuilder";
 import { QueryDocumentsBuilder } from "../../QueryDocuments/QueryDocumentsBuilder";
+import { SPARQLRawResults } from "../../SPARQL/RawResults";
 import { BaseDocumentsRepository } from "../BaseDocumentsRepository";
 import { LDPDocumentsRepositoryTrait } from "./LDPDocumentsRepositoryTrait";
 import { OverriddenMembers, QueryableDocumentsRepositoryTrait } from "./QueryableDocumentsRepositoryTrait";
 
 export interface ExecutableQueryDocumentsRepositoryTrait extends QueryableDocumentsRepositoryTrait {
+
+	/**
+	 * Executes the stored query directly. Returns result in plain JSON Format.
+	 */
 	execute( uri:string ):Promise<JSON>;
+
+	/**
+	 * Executes the stored query and returns the result in a decorated {@link SPARQLRawResults} format.
+	 */
+	executeAsRAWSPARQLQuery( uri:string ):Promise<[ SPARQLRawResults, Response ]>;
+
+	/**
+	 * Modifies the document's stored query
+	 */
 	modifyStoredQuery( uri:string, newStoredQuery:string ):Promise<void>;
```

Comment:
When testing this implementation I ran into some errors. Here's the way I was using the method for review:

```
let domainName = "http://localhost:8083/";
let carbonLDP = new CarbonLDP( domainName );
const C = "https://carbonldp.com/ns/v1/platform#";
const LDP = "http://www.w3.org/ns/ldp#";

let selectQuery = `
	SELECT ?s
	WHERE {
		?s a <${C}Document> .
	}
`;
let executableQueryDocument = CarbonLDP.ExecutableQueryDocument.create( {
	storedQuery: selectQuery
} );

carbonLDP.documents.$createAndRetrieve( "/", executableQueryDocument ).then( executableQueryPersistedDocument => {
	let newQuery = `
		SELECT ?s
		WHERE {
			?s a <${LDP}Resource> .
		}
	`;
	return executableQueryPersistedDocument.$modifyStoredQuery( newQuery );
} ).then( executableQueryPersistedDocument => {
	console.log( executableQueryPersistedDocument.storedQuery );
} ).catch( error => {
	console.error( error.message );
} );
```
This would cause the following error:
`" SELECT ?s WHERE { ?s a <http://www.w3.org/ns/ldp#Resource> . } " is out of scope.`

Also:
```
let domainName = "http://localhost:8083/";
let carbonLDP = new CarbonLDP( domainName );
const C = "https://carbonldp.com/ns/v1/platform#";
const LDP = "http://www.w3.org/ns/ldp#";

let selectQuery = `
	SELECT ?s
	WHERE {
		?s a <${C}Document> .
	}
`;
let executableQueryDocument = CarbonLDP.ExecutableQueryDocument.create( {
	storedQuery: selectQuery
} );

carbonLDP.documents.$createAndRetrieve( "/", executableQueryDocument ).then( executableQueryPersistedDocument => {
	let newQuery = `
		SELECT DISTINCT ?s
		WHERE {
			?s ?p ?o .
		}
	`;
	return executableQueryPersistedDocument.$modifyStoredQuery( newQuery );
} ).then( executableQueryPersistedDocument => {
        // The value returned here is undefined
	console.log( executableQueryPersistedDocument.storedQuery );
} ).catch( error => {
	console.error( error.message );
} );
```
Returns an `undefined` because the method signature is implemented to return a `Promise<void>` value. This method should behave like the `$save` implementation. Also adding an implementation that works as `$saveAndRefresh` would be valuable in this case.